packages/manager-react-components/src/components/datagrid/datagrid.component.tsx

```diff
@@ -37,6 +37,7 @@ import './translations';
 
 export type ColumnSort = TanstackColumnSort;
 export type PaginationState = TanstackPaginationState;
+export type DatagridColumnTypes = FilterTypeCategoriesProps;
```

Comment:
nitpick(blocking): I have below error when i try to import the `DatagridColumnTypes`.

> src/pages/create/index.tsx(26,13): error TS2693: 'DatagridColumnTypes' only refers to a type, but is being used as a value here.

```suggestion
export { FilterTypeCategoriesProps as DatagridColumnTypes };
```

Reply:
👍 corrected the enum has been well exported now